### PR TITLE
PostgreSQL 9.4 (SCL rh-postgresql94)

### PR DESCRIPTION
**Pull request by praiskup**

### CODE REVIEW

**Comment by bparees** [comment]
makefile and readme needs to be updated, see:
https://github.com/openshift/sti-ruby/commit/e188a1d89e1228e698703a9efa7edb438a6fbecb#diff-b67911656ef5d18c4ae36cb6741b7965

**Comment by bparees** [comment]
can you squash the commits?
[test]

File: 9.4/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,53 @@
+FROM centos:centos7
+MAINTAINER  Martin Nagy <mnagy@redhat.com>
```
Comment (mfojtik) [inline 9.4/Dockerfile]: I believe this should be the SCL website

**Comment by bparees** [comment]
@praiskup are you coming back to this PR?

**Comment by bparees** [comment]
@hhorak ?

**Comment by praiskup** [comment]
On Friday 14 of August 2015 19:07:29 Ben Parees wrote:
> can you squash the commits?

Yes ;), but it was intended to have two commits - 'copy' and 'edit', to
allow anybody to see (online) what actually was edited.

**Comment by praiskup** [comment]
Magic [test], while we are on it, can I see the results somewhere?

**Comment by bparees** [comment]
[test]

**Comment by bparees** [comment]
@praiskup test results will show up in the PR you can click details to view the logs on our jenkins.  (once it runs)

**Comment by bparees** [comment]
you're going to want to re-diff this against 9.2 as we made a bunch of changes to 9.2 recently to support password changing.

**Comment by praiskup** [comment]
[test]

**Comment by bparees** [comment]
I think you are not whitelisted to run tests.  let's see if this works:
ok to test

**Comment by bparees** [comment]
lgtm.

**Pull request merged**